packages/component-library/src/Slider/Slider.jsx

```diff
@@ -0,0 +1,63 @@
+import React from "react";
+import { bool, func, node, number } from "prop-types";
+import RcSlider from "rc-slider";
+import "rc-slider/assets/index.css";
+
+import "./slider.css";
+
+const SliderWithTooltip = RcSlider.createSliderWithTooltip(RcSlider);
+
+const Slider = ({
+  min,
+  max,
+  onChange,
+  showTooltip,
+  step,
+  dots,
+  tipFormatter,
+  value,
+  SliderComponent,
+  ...rest
+}) => {
+  return (
+    <span className="civic-slider-container">
+      <SliderComponent
+        dots={dots}
+        min={min}
+        max={max}
+        step={step}
+        value={value}
+        onChange={onChange}
+        {...rest}
```

Comment:
I haven’t used {...rest} but I guess this would allow us to pass in any of the additional props that RCSlider or a custom SliderComponent takes. Seems handy! One trade off is that if someone uses the props that aren’t documented it’d be harder to change the component implementation in the future. I think I like it, but are there other trade offs? 

@mendozaline have you thought about using {...rest} in our map components?